.github/workflows/tilt.yaml

```diff
@@ -0,0 +1,40 @@
+on:
+  pull_request:
+    paths:
```

Comment:
Same [question](https://github.com/operator-framework/operator-controller/pull/319#discussion_r1284155257) as in operator-controller: Maybe `paths-ignore` is more suitable here?